ca/certificate-authority.go

```diff
@@ -298,6 +312,26 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		}
 	}
 
+	// Process requested extensions.  For now, the only extension we support is
+	// TLS Feature [RFC7633], i.e., "Must Staple", and for that we ignore the
+	// value provided by the client and just overwrite it with the specific value
+	// that only requires stapling.
+	//
+	// Other requested extensions are silently ignored.
+	//
+	// XXX(rlb): It might be good to add a generic mechanism for extensions, but
```

Comment:
We don't use XXX comments anymore: https://github.com/letsencrypt/boulder/wiki/Boulder-Development#patch-guidelines (or rather, we use them to mark something we intend to change before merging). Please turn this into a TODO with a link to an issue or email address, or just remove.